static/js/components/ProfileProgressControls.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Button from 'react-mdl/lib/Button';
+
+import { saveProfileStep } from '../util/profile_edit';
+
+export default class ProfileProgressControls extends React.Component {
+  static propTypes = {
+    nextUrl: React.PropTypes.string,
+    prevUrl: React.PropTypes.string,
+    isLastTab: React.PropTypes.bool,
+    saveProfile: React.PropTypes.func.isRequired,
```

Comment:
right, thanks for the reminder. there's just something very opaque about all this. finding out where `saveProfile` comes from and where it's used involves a bunch of different files. `saveProfileStep` itself relies on a side effect of sorts to work properly - it depends on `saveProfile` and `profile` to be available on `this.props`. maybe that is an understood pattern in our codebase and/or the React community itself. still, i would think an API like this would make so much more sense: `function saveProfileStep(profile, isLastStep=false)`. in the body of that function, you'd call `profile.save()`. this would involve creating a new class that encapsulates profile data and functionality. i'm not sure how heavy a lift that would be, but i think it's definitely preferable to the current approach.

anyway, that could be a future refactor. for now, this PR definitely improves what we have